README.md

```diff
@@ -1,14 +1,13 @@
 # Boiler Room Builder
 
-[![Build status](https://badge.buildkite.com/35edf858022bf6c8ec20dc8a3433348f4a268d772991e2c913.svg)](https://buildkite.com/everyday-hero/boiler-room-runner-tests)
+[![Build status](https://badge.buildkite.com/3b53417ee3fb24b145098c129e6b31cc2ddc42d39d98ab0b52.svg?style=flat-square)](https://buildkite.com/everyday-hero/boiler-room-builder-tests)
 
-A tool-chain for universal react apps
+Create React apps without writing your own build configuration. With little effort you will be able to `build`, `test`, `lint` and `serve` like a pro.
```

Comment:
"You write the React app, Boiler Room Builder (BRB) handles the build configuration."